packages/compiler-ssr/__tests__/ssrElement.spec.ts

```diff
@@ -337,6 +337,39 @@ describe('ssr: element', () => {
         `)
     })
 
+    test('custom dir with v-text', () => {
```

Comment:
These test cases can pass without the following changes. Maybe the test cases should be written in `packages/server-renderer/__tests__/ssrDirectives.spec.ts` to verify the rendering result.

Reply:
Hi @edison1105! All four added tests fail without my changes in `packages/compiler-ssr/src/transforms/ssrTransformElement.ts`. I chose the file for these tests by following a similar commit example: [79602f9](https://github.com/vuejs/core/commit/79602f9ecd9559954f844774a90286305b13e056).

If you think it’s necessary, I can also add `v-text` tests to `packages/server-renderer/__tests__/ssrDirectives.spec.ts`. However, I didn’t see any other tests for `v-text` or `v-html` in that file, and tests for the behavior of these directives in SSR mode currently seem to be only in `packages/compiler-ssr/__tests__/ssrElement.spec.ts`.

Reply:
Ah, It seems I made a mistake.
Thanks for your contribution.